2024-07-16 07:05:37,179 ERROR Бот упал
2024-07-16 07:08:09,037 ERROR Бот упал
2024-07-16 07:08:12,857 ERROR Бот упал
2024-07-16 07:13:02,642 ERROR Бот упал. exceptions must derive from BaseException
2024-07-16 09:40:37,503 ERROR Бот упал. [Errno 2] No such file or directory: 'tempArh/f_bcd397b4-d353-461c-b704-75e3347569de.zip'
2024-07-16 10:33:54,500 ERROR Бот упал. list.append() takes exactly one argument (7 given)
2024-07-16 16:06:12,841 ERROR Бот упал. HTTPSConnectionPool(host='api.telegram.org', port=443): Read timed out. (read timeout=25)
2024-07-17 12:27:06,854 ERROR Бот упал. HTTPSConnectionPool(host='api.telegram.org', port=443): Max retries exceeded with url: /bot6597327676:AAHXa04PV0HRvBBTDzBOTI8WwlfEDM9V6Tk/getUpdates?offset=1&timeout=20 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002AA8DEA8350>: Failed to resolve 'api.telegram.org' ([Errno 11001] getaddrinfo failed)"))
2024-07-17 14:29:13,315 ERROR Бот упал. HTTPSConnectionPool(host='api.telegram.org', port=443): Max retries exceeded with url: /bot6597327676:AAHXa04PV0HRvBBTDzBOTI8WwlfEDM9V6Tk/getUpdates?offset=1&timeout=20 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x0000022ADC698CB0>: Failed to resolve 'api.telegram.org' ([Errno 11001] getaddrinfo failed)"))
